api/src/app.ts

```diff
@@ -17,6 +17,16 @@ import { getLogger } from './utils/logger';
 
 const defaultLog = getLogger('app');
 
+/**
+ * Extending express request type.
+ *
+ */
+declare module 'express-serve-static-core' {
+  interface Request {
+    files: Express.Multer.File[];
```

Comment:
Im uncertain if this should be optional or not.

The 'files' property only exists on multipart/form-data endpoints, but those endpoints now have the correct OpenAPI spec where the 'files' property will be defined.